### ISSUE TITLE
Fix mobilito tutorial

### ISSUE THREAD

**Issue by JeffAbrahamson** [transport-nantes/tn_web#1202]
* Simplify, user testing says we were too complicated.
* Make it the only thing visible when it happens: users get distracted looking at lower-down text, footers, etc.  Probably make it modal.
* Simplify visual: it's too busy and doesn't adequately tutor.
* The only state we should maintain beyond the tutorial is a db flag that the user saw it (and when).
* We should have a tutorial event table for sending / composing reminders.  That should be a separate issue, however, once this is done.
